### ISSUE TITLE
Asciidoc Preview layout broken in Android Studio

### ISSUE THREAD

**Issue by j-s-a-m** [asciidoctor/asciidoctor-intellij-plugin#531]
# Issue:
The Asciidoc preview is looking broken in Android Studio, whereas in IntelliJ IDEA, it's working fine.

I created a minimal Android project from scratch (just new > Project > no Activity) and created a simple Asciidoc file (issue.adoc). The preview in **Android Studio** (4.0.1) looks like this:
![Asciidoc_preview_AndroidStudio](https://user-images.githubusercontent.com/6809386/89763019-ed370180-daf1-11ea-92d3-59674adc8dda.png)

When I open the same project in **IntelliJ IDEA** (2019.3 , on the same machine), the preview looks much better: 
![Asciidoc_preview_IntelliJIdea](https://user-images.githubusercontent.com/6809386/89763016-ec9e6b00-daf1-11ea-92a9-dec3db546fb9.png)

I double-checked this with an older Version of Android Studio (3.6.2. / AI-192.7142.36.36.6308749) and the according older version of the Asciidoc plugin, but the effect is the same.

Is this a bug in the plugin? Or just some configuration issue on my side?

# Versions used

AsciiDoc Plugin Version: 0.30.85

IDE Name and Version: Android Studio 4.0.1
Build #AI-193.6911.18.40.6626763, built on June 25, 2020
Runtime version: 1.8.0_242-release-1644-b3-6222593 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.4.0-42-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 32
Registry: ide.new.welcome.screen.force=true
Non-Bundled Plugins: org.jetbrains.kotlin, com.google.services.firebase, com.intellij.marketplace, com.thoughtworks.gauge, de.mobilej.adc.plugin.id, org.asciidoctor.intellij.asciidoc

**Comment by ahus1** [comment]
Looking at the screen shot the preview in Android Studio uses the Swing preview browser while in IntelliJ it uses the JavaFX preview.

The JavaFX preview you see in IntelliJ is more advanced and looks a lot nicer.  

It seems that Google doesn't pre-bundle JavaFX in the runtime for Android Studio. Have a look at https://github.com/asciidoctor/asciidoctor-intellij-plugin/wiki/JavaFX-preview -- the "Choose Runtime" might be a good way to change to a JetBrains JDK with JavaFX support. 

As I am not an Android developer, I don't know if Android Studio works better with JetBrains JDK11 or JetBrains JDK8. Please let me know, and I can update the documentation.

**Comment by ahus1** [comment]
PS: See [configuring preview](https://intellij-asciidoc-plugin.ahus1.de/docs/users-guide/recommended-settings.html#_configuring-preview) for more information on how to configure the preview.

**Comment by j-s-a-m** [comment]
Although I was using the JetBrains Java Runtime bundled with the latest version of Android Studio 4.0.1, Java FX previews were not available. 

So I downloaded a more recent JetBrains Runtime (jbr-11_0_8-linux-x64-b1031.1.tar.gz) from https://confluence.jetbrains.com/display/JBR/JetBrains+Runtime , installed it and registered it the "alternatives system".

Then I used the Android Studio plugin "Choose Runtime" to conveniently switch to that runtime. Now the Java FX preview is available in Android Studio and Asciidoc previews are looking great!

@ahus1 Thanks for your support!
